src/luascript.cpp

```diff
@@ -10825,6 +10827,29 @@ int LuaScriptInterface::luaPlayerGetIdleTime(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaSendCreatureSquare(lua_State* L)
+{
+	// player:sendCreatureSquare(creature, color)
+	Player* player = getUserdata<Player>(L, 1);
+	if (!player) {
+    	lua_pushnil(L);
+    	return 1;
+	}
+
+	Creature* creature = getUserdata<Creature>(L, 2);
+	if (!creature) {
+		reportErrorFunc(L, getErrorDesc(LUA_ERROR_CREATURE_NOT_FOUND));
+    	lua_pushboolean(L, false);
+    	return 1;
+	}
+
+	uint32_t color = getNumber<uint32_t>(L, 3, 0);
```

Comment:
unnecessary temporary.
The `getNumber` method already applies the conversion, use it directly as a parameter in the call,